packages/block-editor/src/components/url-popover/image-url-input-ui.js

```diff
@@ -227,16 +227,14 @@ const ImageURLInputUI = ( {
 				checked={ linkTarget === '_blank' }
 			/>
 			<TextControl
-				// TODO: Switch to `true` (40px size) if possible
-				__next40pxDefaultSize={ false }
+				__next40pxDefaultSize
```

Comment:
Image block ▸ "Link" in block toolbar ▸ Down caret ("Link settings") dropdown in the popover.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/1f020118-660f-4d9d-ab85-89165fb35894" alt="Image block URL popover, before" width="512">|<img src="https://github.com/user-attachments/assets/cb1775d5-fa0b-475f-9af3-9f720b1bb21e" alt="Image block URL popover, after" width="512">|